src/styles/index.scss

```diff
@@ -1,3 +1,14 @@
+@import "./blocks/consts.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  line-height: 14px;
+  font-size: 10px;
+  color: $text-root-color;
+}
```

Comment:
hint: you can use `index.scss` only for imports all style-files and move common styles for `body`, `html`,  ... etc to separate `main.scss` or `common.scss` file as well.